config/install/field.field.media.image.field_islandora_image.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
     - media_entity.bundle.image
   module:
     - image
+    - islandora_image
```

Comment:
this dependency needs to be nested in a enforced block.  the entire chunk should look like this: 
```
dependencies:
  config:
    - field.storage.media.field_islandora_image
    - media_entity.bundle.image
  module:
    - image
  enforced:
    module:
      - islandora_image
```